README.md

```diff
@@ -1,56 +1,57 @@
-﻿# Bootstrap WinJS
-﻿
-http://bootstrap-winjs.azurewebsites.net/
-
-## Development
-
-### Install Node.js
-
-http://nodejs.org/download/
-
-Make sure Node.js is available in your path.
-
-### Install Grunt
-
-```
-npm install -g grunt-cli
-```
-
-### Clone the repo
-
-```
-git clone https://github.com/winjs/bootstrap-winjs.git
-```
-
-You should have a `bootstrap-winjs` folder.
-
-### Installing npm packages
-
-To install the required `npm` packages:
-
-```
-cd bootstrap-winjs
-npm install
-```
-
-## Building bootstrap-winjs
-
-Just run:
-
-```
-grunt
-```
-
-This will compile the generated files into the `dist` folder.
-
-### Launching a local server
-
-Just run this task to display the demo pages in your browser:
-
-```
-grunt server
-```
-
-Launch this URL in your brower: http://localhost:9001/
-
-The changes you make to the Sass files `*.scss` or source documentation files `*.hbs` will be automatically recompiled. Just refresh your browser window.
+﻿# Winstrap
+##### The official Bootstrap theme for Windows
+﻿
+http://bootstrap-winjs.azurewebsites.net/

```

Comment:
Nice. What do we do about the repository name itself? Rename? Or start a new one and use this one to point to it?